airbyte-integrations/connectors/source-sendgrid/manifest.yaml

```diff
@@ -640,6 +640,8 @@ definitions:
           path: "{{stream_slice['url']}}"
           url_base: ""
           http_method: GET
+        download_extractor:
+          type: ResponseToFileExtractor
```

Comment:
There was a subtle change in this PR: https://github.com/airbytehq/airbyte-python-cdk/pull/50/files#diff-1d9bc4ca384e5c00867da05e9db9d919aba908c59bb73aab83883b9ed20def05L2040

Where we no longer automatically used the `ResponseToFileExtractor` if `download_extractor` was not defined. Instead we defaulted to the JsonDecoder which broke sendgrid. Since this was the only use of AsyncRetriever, its easy enough to just fix it here instead of a full breaking change.